.github/workflows/wheels.yml

```diff
@@ -199,3 +199,61 @@ jobs:
           source ci/upload_wheels.sh
           set_upload_vars
           upload_wheels
+
+  # Build Pyodide wheels and upload them to Anaconda.org
+  # NOTE: this job must be kept in sync with the emscripten job in unit-tests.yml.
+  # The difference is that this does not run the tests after building the wheels.
+  build_pyodide_wheels:
+    name: Build Pyodide wheels
+    # Run on schedule, manual (workflow dispatch), and push events
+    # Skip on PRs, because these are running in the unit tests job already
+    if: >-
+      (github.event_name == 'schedule') ||
+      (github.event_name == 'workflow_dispatch') &&
+      (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v') && ( ! endsWith(github.ref, 'dev0')))
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout pandas Repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python for Pyodide
+        id: setup-python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11.3'
+
+      - name: Set up Emscripten toolchain
+        uses: mymindstorm/setup-emsdk@v14
+        with:
+          version: '3.1.46'
+          actions-cache-folder: emsdk-cache
+
+      - name: Install pyodide-build
+        run: pip install "pyodide-build==0.25.1"
+
+      - name: Build pandas for Pyodide
+        # Here, upload_wheels.sh requires artifacts to be in ./wheelhouse/
+        run: |
+          mkdir -p wheelhouse
+          pyodide build --outdir wheelhouse
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: pyodide_wheel
+          path: ./wheelhouse/*.whl
+

```

Comment:
I have included this step in case it is needed to inspect the binary and do some debugging – happy to remove it based on what is suggested.

Reply:
I think it might be better to fuse the two jobs now (since there doesn't look like there's any pyodide specific logic in the job), and we'd probably want the triggers to be sync'ed up across the build jobs.

Reply:
Let me try that – I think the only special logic is the `CIBW_PLATFORM: pyodide` environment variable, which I can just put in an `- include:`. Let me push this in a moment.

Reply:
Addressed in 04cee9a1cea29ca511fc663af1295e36b9b59db1.